### PR TITLE
ci: Pin compatible aws-sdk-core version for aws_30.gemfile

### PR DESCRIPTION
**Pull request by Ferenc-**

### CODE REVIEW

**Review by pvital — APPROVED** [review]
It looks good to me.

File: gemfiles/aws_30.gemfile [hunk 1]
```diff
@@ -11,6 +11,7 @@ gem "webrick"
 gem "puma"
 gem "rack-test"
 gem "simplecov", "~> 0.21.2"
+gem "aws-sdk-core", "= 3.191.6"
```
Comment (GSVarsha) [inline gemfiles/aws_30.gemfile:14]: Can we mention somewhere (maybe in the commit message) the reason why we use a strict version constraint for `aws-sdk-core` gem?

Comment (Ferenc-) [reply gemfiles/aws_30.gemfile:14]: Wrote in the commit message. Please check.

Comment (GSVarsha) [reply gemfiles/aws_30.gemfile:14]: Thank you!

**Pull request merged**